tests_e2e/tests/lib/agent_log.py

```diff
@@ -65,30 +65,12 @@ def from_dictionary(dictionary: Dict[str, str]):
 
     @property
     def timestamp(self) -> datetime:
-        # Extension logs may follow different timestamp formats
-        # 2023/07/10 20:50:13.459260
-        ext_timestamp_regex_1 = r"\d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2}[.\d]+"
-
-        # 2023/07/10 20:50:13 OR 2023/07/10 20:50:1
-        # 2023/07/10 20:50 OR 2023/07/10 20:5
-        ext_timestamp_regex_2 = r"(\d{4}/\d{2}/\d{2} \d{2}:\d{1,2})(?::(\d{1,2}))?"
-
-        if re.match(ext_timestamp_regex_1, self.when):
+        # The timestamps emitted by the Agent can follow two formats: 2025-11-05T23:52:48.037713Z (newer) or 2025-11-05T23:52:48.790673Z (older)
```

Comment:
Is this comment correct? Looks like both formats are the same

Reply:
thanks; copy-n-paste error. should be 2025/11/07 00:56:06.858951 (older)